README.rst

```diff
@@ -33,21 +44,30 @@ While viewing documentation, you can easily propose a change by selecting **sugg
 The page you are viewing then opens in GitHub, in edit mode. When you save your edits, create a new branch, commit your changes,
 and create a new Pull Request, to have your changes reviewed by the Open edX team.
 
-Build and Test Documentation
-****************************
+Build and Test Documentation Locally
+************************************
+
+For developers comfortable pulling down the repository locally and making changes (see `GitHub Flow`_), you can build the documentation directly on your machine.
 
 To build documentation, inside the root folder, run:
 
 .. code-block:: bash
 
+    make clean
     make html
 
-Sphinx should build the HTML files locally.
+Sphinx should build the HTML files locally. To serve the docs locally, run
+
+.. code-block:: bash
+
+    make serve_docs
+
+This is a background process, and it will continue updating the docs as you edit them. However if you make substantive changes, you may need to kill the process and run ``make clean``/``make html`` again. Unfortuantely this is a big project and ``make html`` can take 3-4 minutes to run.
```

Comment:
```suggestion
This is a background process, and it will continue updating the docs as you edit them. However if you make substantive changes, you may need to kill the process and run ``make clean``/``make html`` again. We're fortunate to have a lot of docs - but this means that ``make html`` can take 3-4 minutes to run.
```